### ISSUE TITLE
Some permissions are not documented

### ISSUE THREAD

**Issue by Cussa** [WeblateOrg/weblate#1900]
Some of the permissions are not documented. 

Example:
trans | check | Can add check
trans | check | Can change check
trans | check | Can delete check
trans | check | Can ignore check results
trans | source | Can edit check flags

**Comment by nijel** [comment]
These are not used by Weblate, but are by default created by Django and there is AFAIK no way to tell Django to not generate these.

The whole permission system will be anyway soon rewritten from scratch (see https://github.com/WeblateOrg/weblate/issues/1798), so all these Django built in permissions will be gone.

**Comment by Cussa** [comment]
I get it. So, let me rephrase myself.
I'm configuring the permission on a way that I have a group of translators and only them are able to change the translation itself. This means that if there is some failing check (like "Unchanged translation"), only people from this group will be able to discard (click on the X) the message. But I didn't get which permission is giving this privilege.

**Comment by nijel** [comment]
The Weblate defined permission are described here: https://docs.weblate.org/en/latest/admin/access.html#extra-privileges (still Weblate does use some of the standard ones, what makes current setup pretty confusing). Apparently this one is missing, so I'm adding it right now.

The one you're looking for is:

>  trans | check | Can ignore check results

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has just been fixed. 

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).

**Comment by Cussa** [comment]
So, maybe there is a bug. (or maybe I'm understanding it wrong...)
Because I'm configuring this permission, but I'm still not able to ignore the check fail.

Using a superuser, I saw it like this:
![image](https://user-images.githubusercontent.com/912710/37294048-4837007e-2615-11e8-930d-3735eae3f7d5.png)

But when I use a user from the group of the translators, the X doesn't appear to me.
![image](https://user-images.githubusercontent.com/912710/37294170-7ee410da-2615-11e8-8dff-683f6f318ddf.png)

Here is how I configured it.

Group: ABC Translators
- trans | check | Can ignore check results
- trans | suggestions | Can accept suggestions
- trans | suggestions | Can delete suggestions
- trans | translations | Can add translations
- trans | translations | Can delete translations
- trans | translations | Can overwrite with translation upload
- trans | translations | Can upload translation
- trans | translations | Can use machine translation
- trans | unit | Can save translation

Group: Users
- trans | comment | Can add comment
- trans | Project | Can access project
- trans | suggestion | Can add suggestion
- trans | suggestion | Can vote for suggestion

ACL: [Project 1 - Language ABC]
- trans | check | Can ignore check results
- trans | suggestions | Can accept suggestions
- trans | suggestions | Can delete suggestions
- trans | translations | Can add translations
- trans | translations | Can delete translations
- trans | translations | Can overwrite with translation upload
- trans | translations | Can upload translation
- trans | translations | Can use machine translation
- trans | unit | Can save translation

User 1 - Groups:
- ABC Translators
- Users

(By the way, I was using the right permission, because it was my first choice! 😄 )

**Comment by nijel** [comment]
The ACL on both project and language currently do not work as expected in some situations, this is probably outcome. Just to make sure it's configured correctly on your side: does the ACL have ABC Translators group assigned?

This breakage is partly documented in https://github.com/WeblateOrg/weblate/issues/1687 and is one of reasons for https://github.com/WeblateOrg/weblate/issues/1798. The ACL thing we're using was designed for different use case and the way we're using it now is hitting limits...

**Comment by Cussa** [comment]
> Just to make sure it's configured correctly on your side: does the ACL have ABC Translators group assigned?
Yes. I reconfirmed right now to be sure.

So, maybe this is one of the situations where it's not behaving correct.

**Comment by Cussa** [comment]
@nijel , just more one thing: as there is some idea about rewrite this thing about permission, it will be wonderful if you can create a ACL (or the new model), but related to several languages.

For example, I want to create a group "English Translators" that could translate any english language (US, UK, etc). Today, I have one Group (English Translators) and many ACLs related to this Groups (en_US, en_UK, etc)

**Comment by nijel** [comment]
This should work just fine in currently planned design. Anyway it's better to discuss in https://github.com/WeblateOrg/weblate/issues/1798
